src/pages/home/JuicyPicksSection/JuicyPicksSection.tsx

```diff
@@ -17,9 +15,7 @@ export function JuicyPicksSection() {
           <Trans>Peep our selection of top trending projects this month.</Trans>
         }
       />
-      <Row gutter={DEFAULT_HOMEPAGE_GUTTER}>
-        {isMobile ? <JuicyPicksMobile /> : <JuicyPicks />}
-      </Row>
+      {isMobile ? <JuicyPicksMobile /> : <JuicyPicks />}
```

Comment:
How different is mobile to desktop? Do we really need 2 components? is there a solution where one component will work?

Reply:
Definitely need two components I think. Mobile is just normal slider but desktop is this totally different layout: 

<img width="234" alt="Screen Shot 2023-04-13 at 7 14 34 am" src="https://user-images.githubusercontent.com/96150256/231660185-1a492518-33ff-4fd5-a71b-d76e69582f8c.png">

<img width="547" alt="Screen Shot 2023-04-13 at 7 14 40 am" src="https://user-images.githubusercontent.com/96150256/231660129-a685a4ba-b2c0-41b5-b733-91f9f927412a.png">

Reply:
I think we can merge these 2 components:
- on mobile, hide the left col
- on mobile, nowrap left col, horizontal scroll